### DISCUSSION TITLE
the  yalmiptime and solvertime

### DISCUSSION THREAD

**Discussion by lzc100** [yalmip/YALMIP#990] [General discussion] [answered]
Hi Johan,
I am confused about whether the yamliptime should be included in the evaluation of solution efficiency, or  only solvertime should be considered. In other words, if I build the model in the native way of solver, is there be no modeling time when solving the problem.
Thanks

**Comment by johanlofberg** [comment, marked as answer]
Modelling time is completely unrelated to solver efficiency. It is not the solvers fault that yalmip might have large overhead.

**Reply by lzc100** [reply]
So if I want to compare the solution efficiency of the interior point method I wrote in MATLAB with yamlip , I should only compare it with solvertime, is it right?

**Reply by johanlofberg** [reply]
You don't compare it with yalmip, you compare it with the solver you are using.

**Reply by lzc100** [reply]
got it, thanks a lot
